ironfish-cli/src/commands/chain/readd-block.ts

```diff
@@ -18,7 +18,6 @@ export default class ReAddBlock extends IronfishCommand {
   static args = [
     {
       name: 'hash',
-      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
```

Comment:
Won't this break the system if the user adds extra whitespace here?

Reply:
oclif ignores whitespace in parsing unless you escape the characters or wrap them in quotes, so `ironfish chain:readd-block dead...beef` is the same as `ironfish chain:readd-block                      dead...beef`